stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3821 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.                win_perc = num_win / num_total if num_total &gt; 0 else np.nan
</pre><p>This isn't a percentage (either multiply by 100 or rename the column/variable) - seems to be referred to as win rate later on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Renamed